features/integration/simulate.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Simulating transactions
+  Background:
+    Given an algod v2 client
+    And a kmd client
+    And wallet information
+    And suggested transaction parameters from the algod v2 client
+    And I create a new transient account and fund it with 10000000 microalgos.
+    And I make a transaction signer for the transient account.
+
+  @simulate
+  Scenario Outline: Simulating successful payment transactions
+    Given default transaction with parameters <amt> "<note>"
+    When I get the private key
+    And I sign the transaction with the private key
+    And I simulate the transaction
+    Then the simulation should succeed without any failure message
+
+    Examples:
+      | amt     | note         |
+      | 0       | X4Bl4wQ9rCo= |
+      | 1234523 | X4Bl4wQ9rCo= |
+
+  @simulate
+  Scenario: Simulating duplicate transactions in a group and overspending errors
+    Given a new AtomicTransactionComposer
+    When I build a payment transaction with sender "transient", receiver "transient", amount 100001, close remainder to ""
+    And I create a transaction with signer with the current transaction.
+    And I add the current transaction with signer to the composer.
+    Then I simulate the current transaction group with the composer
```

Comment:
Similar question. Should we anticipate the following step, and therefore rename this one?

```cucumber
Then I simulate-signed the current transaction group with the composer
```